frontend/src/Editor/Viewer/MobileNavigationMenu.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import _ from 'lodash';
+// eslint-disable-next-line import/no-unresolved
+import { slide as MobileMenu } from 'react-burger-menu';
+import { DarkModeToggle } from '@/_components/DarkModeToggle';
+import Header from './Header';
+import Cross from '@/_ui/Icon/solidIcons/Cross';
+
+const MobileNavigationMenu = ({ pages, switchPage, currentPageId, darkMode, changeDarkMode }) => {
+  const [hamburgerMenuOpen, setHamburgerMenuOpen] = React.useState(false);
```

Comment:
```suggestion
  const [hamburgerMenuOpen, setHamburgerMenuOpen]
 = useState(false);
```

We can follow importing the `useState` and use it